### DISCUSSION TITLE
vuejs3 and flowbite + datepicker

### DISCUSSION THREAD

**Discussion by gfernandez-me** [themesberg/flowbite#97] [Q&A]
hello,

for me the datepicker is not working

```
<input
      id="date-picker"
      ref="date-picker"
      datepicker
      type="text"
      class="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 pl-10 text-gray-900 focus:border-blue-500 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-blue-500 dark:focus:ring-blue-500 sm:text-sm"
      placeholder="Select date" />
```
should I import the datepicker on my main.js?
the "@themesberg/tailwind-datepicker": "^1.1.0", is already on my package.json file

**Comment by JohnHasmie** [comment]
You can import in specific/needed component

import Datepicker from 'flowbite-datepicker/Datepicker';

const datepickerEl = document.getElementById('date-picker');
new Datepicker(datepickerEl);
